docs/pages/input_schema/input_types.md

```diff
@@ -250,3 +250,6 @@ If multiple types are defined, the user can choose which type to use.
 ```
 
 This input supported either a number or a string as valid inputs.
+
+## oneOf
+`oneOf` is another way of allowing multiple types for an input, but is more complex to use and offers more flexibility. For a more detailed guide, [see here](oneof)
```

Comment:
not in this case, because the markdown is used for a static site generator. when you are viewing this page it is `/input-schema`.  this link will  take it to `/input-schema/oneof`, which  is the correct url for the oneof page